package-lock.json

```
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.70.1",
+      "version": "0.70.17",
```

Revert the package version back to the one in base branch! 
```suggestion
      "version": "0.70.1",
```